docs/modules/sql/pages/sql-reflection-configuration.adoc

```diff
@@ -0,0 +1,96 @@
+= Configuring Reflection
+:description: Clusters can be configured to restrict the capability of creating objects through reflection during SQL execution.
+
+Clusters can be configured to restrict the capability of creating objects through reflection during SQL execution.
+
+== Disallowing Reflection of Untrusted Java Classes
+
+To prevent Hazelcast from constructing objects of arbitrary classes through reflection, you have the option to permit or deny the reflection for Java classes based on class name, package name, or prefix using the java-reflection-filter parameter.
+
+The following filtering rules are used when the objects are constructed through reflection:
+
+* When the `whitelist` option is empty:
+** If the object's class name or package name is blacklisted, reflection fails.
+** Otherwise, reflection is allowed.
+* When the `whitelist` option is provided:
```

Comment:
consider "populated" instead of "provided"

Reply:
b78116e8d8a8fd65fb1046577fd0c5dd0fa0e5a2